api/src/main/scala/com/rasterfoundry/granary/Server.scala

```diff
@@ -78,7 +86,13 @@ object ApiServer extends IOApp {
         .httpRoutes(false, false) {
           CORS(
             Router(
-              "/api" -> (helloRoutes <+> modelRoutes <+> predictionRoutes <+> docRoutes)
+              "/api" -> (
+                Auth.customAuthMiddleware(
+                  helloRoutes <+> modelRoutes <+> predictionRoutes <+> docRoutes,
```

Comment:
I don't think `docRoutes` should be here because it prevents putting the doc URL into open API tools. It should work with something like:

```Auth.customAuthMiddleware(helloRoutes <+> modelRoutes <+> predictionRoutes, authConfig, transactor) <+> docRoutes)```

Reply:
We tried that out in pairing I think? That it didn't work is where the comment about "Both the auth and docs middleware want to be last" came from

Reply:
Hmm, on this branch I just did: 

```
(Auth.customAuthMiddleware(helloRoutes <+> modelRoutes <+> predictionRoutes, authConfig, transactor) <+> docRoutes)
```

and it compiles.

Reply:
Ah ok, I see what you mean now - the request gets rejected before it can try to match on the docs route.

I would propose this then for the router:
```
          Router(
              "/" -> docRoutes,
              "/api" -> Auth.customAuthMiddleware(
                helloRoutes <+> modelRoutes <+> predictionRoutes,
                authConfig,
                transactor
              )
            )
```